Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```
@@ -66,6 +69,11 @@
       - id: LPPMagazinePistolPF20lethal
       - id: LPPMagazinePistolPF20Rubber
         prob: 0.5 #Кибербуллинг метаюзеров :D
+      # -< DarkSide
+      - id: LPPBoxFolderSecretRed
+      - id: LPPBoxFolderSecDoc
+        prob: 0.5
+      # - >
```

yamllint

[warning] comment not indented like content

(comments-indentation)